checkout-extension/src/Checkout.liquid

```diff
@@ -1,33 +1,38 @@
 {%- if flavor contains "react" -%}
-import React from 'react';
 import {
+  Banner,
   useApi,
   useTranslate,
   reactExtension,
-  Banner,
 } from '@shopify/ui-extensions-react/checkout';
 
-export default reactExtension('purchase.checkout.block.render', () => <App />);
+export default reactExtension(
+  'purchase.checkout.block.render',
+  () => <Extension />,
+);
 
-function App() {
-  const { extension: { target } } = useApi();
```

Comment:
IMO, nested destructuring is a little overwhelming for folks who aren't super familiar with JavaScript (and even some that are), because it's not super common to see and doesn't always have a matching concept in other languages. I restricted myself to a single level of destructuring in these examples.